stream/file.go

```diff
@@ -0,0 +1,28 @@
+package stream
+
+import (
+	"io"
+	"os"
+)
+
+// File is a source for streams which is backed by a file on the file system.
+// Concurrency and safety is provided by the file system, hence no need for
+// mutex/locks. The write method is provided for free by *os.File.
+type File struct {
```

Comment:
What is the purpose of this struct and is it the final version (or will there be more code in the future)? If this is the final version: Why not just using the `os.File` struct?